apps/src/componentLibrary/chips/Chips.story.tsx

```diff
@@ -0,0 +1,138 @@
+import React, {useState} from 'react';
+import {Meta, Story} from '@storybook/react';
+
+import {Chips, ChipsProps} from './index';
+
+export default {
+  title: 'DesignSystem/Chips Component',
+  component: Chips,
+} as Meta;
+
+//
+// TEMPLATE
+//
+// This is needed to fix children type error (passing string instead of React.ReactNode type)
+// eslint-disable-next-line
+const SingleTemplate: Story<ChipsProps> = args => {
+  const [values, setValues] = useState([]);
```

Comment:
does parameterizing `useState` as `useState<string[]>` allow you to remove the cast below?

Reply:
It is, thanks for noticing!